Library/Formula/kapacitor.rb

```diff
@@ -0,0 +1,189 @@
+    url "https://github.com/shurcooL/markdownfmt.git",
+      :revision => "c6ad3c69d14e5089336af1e41e595f0826df1b82"
+  end
+
+  go_resource "github.com/dustin/go-humanize" do
+    url "https://github.com/dustin/go-humanize.git",
+      :revision => "64dbdae0d393b7d71480a6dace78456396b55286"
+  end
+
+  go_resource "github.com/mattn/go-runewidth" do
+    url "https://github.com/mattn/go-runewidth.git",
+      :revision => "d96d1bd051f2bd9e7e43d602782b37b93b1b5666"
+  end
+
+  go_resource "github.com/russross/blackfriday" do
+    url "https://github.com/russross/blackfriday.git",
+      :revision => "300106c228d52c8941d4b3de6054a6062a86dda3"
+  end
+
+  go_resource "github.com/shurcooL/go" do
+    url "https://github.com/shurcooL/go.git",
+      :revision => "ee4d3ba2f39466dd17fc55dd260e89b8963640f8"
+  end
+
+  go_resource "github.com/shurcooL/sanitized_anchor_name" do
+    url "https://github.com/shurcooL/sanitized_anchor_name.git",
+      :revision => "10ef21a441db47d8b13ebcc5fd2310f636973c77"
+  end
+
+  def install
+    ENV["GOPATH"] = buildpath
+    kapacitor_path = buildpath/"src/github.com/influxdb/kapacitor"
+    kapacitor_path.install Dir["*"]
+    revision = `git rev-parse HEAD`
+
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    cd kapacitor_path do
+      system "go", "install",
+             "-ldflags", "-X main.version=#{version} -X main.commit=#{revision}",
+             "./..."
+    end
+
+    inreplace kapacitor_path/"etc/kapacitor/kapacitor.conf" do |s|
+      s.gsub! "/var/lib/kapacitor", "#{var}/kapacitor"
+      s.gsub! "/var/log/kapacitor", "#{var}/log"
+    end
+
+    bin.install "bin/kapacitord"
+    bin.install "bin/kapacitor"
+    etc.install kapacitor_path/"etc/kapacitor/kapacitor.conf" => "kapacitor.conf"
+
+    (var/"kapacitor/replay").mkpath
+    (var/"kapacitor/tasks").mkpath
+  end
+
+  plist_options :manual => "kapacitord -config #{HOMEBREW_PREFIX}/etc/kapacitor.conf"
+
+  def plist; <<-EOS.undent
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+      <dict>
+        <key>KeepAlive</key>
+        <dict>
+          <key>SuccessfulExit</key>
+          <false/>
+        </dict>
+        <key>Label</key>
+        <string>#{plist_name}</string>
+        <key>ProgramArguments</key>
+        <array>
+          <string>#{opt_bin}/kapacitord</string>
+          <string>-config</string>
+          <string>#{HOMEBREW_PREFIX}/etc/kapacitor.conf</string>
+        </array>
+        <key>RunAtLoad</key>
+        <true/>
+        <key>WorkingDirectory</key>
+        <string>#{var}</string>
+        <key>StandardErrorPath</key>
+        <string>#{var}/log/kapacitor.log</string>
+        <key>StandardOutPath</key>
+        <string>#{var}/log/kapacitor.log</string>
+      </dict>
+    </plist>
+    EOS
+  end
+
+  test do
+    assert_match /Kapacitor/, shell_output("kapacitor version")
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!